docs/pages/admin-guides/access-controls/idps/saml-microsoft-entra-external-id.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Access Azure Portal and CLI 
+description: Access Azure Portal and CLI by authenticating with Teleport SAML IdP
+---
+
+This guide explains how to integrate Teleport SAML IdP with [Microsoft Entra External ID](https://learn.microsoft.com/en-us/entra/external-id/)
```

Comment:
```suggestion
This guide explains how to integrate the Teleport SAML IdP with [Microsoft Entra External ID](https://learn.microsoft.com/en-us/entra/external-id/)
```

Reply:
Fixed all the missing `the` article in https://github.com/gravitational/teleport/pull/54019/commits/a83c8cbfdf38d53134861e3dc08900bd1dd36c4c